x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/cleaner/AbstractIndicesCleanerTestCase.java

```diff
@@ -196,48 +195,48 @@ protected void createWatcherHistoryIndex(final DateTime creationDate) {
     /**
      * Creates a watcher history index from the specified version.
      */
-    protected void createWatcherHistoryIndex(final DateTime creationDate, final String version) {
-        final DateTimeFormatter formatter = DateTimeFormat.forPattern("YYYY.MM.dd").withZoneUTC();
-        final String index = ".watcher-history-" + version + "-" + formatter.print(creationDate.getMillis());
+    protected void createWatcherHistoryIndex(final ZonedDateTime creationDate, final String version) {
+        DateFormatter formatter = DateFormatter.forPattern("YYYY.MM.dd").withZone(ZoneOffset.UTC);
```

Comment:
does it make sense to add a test for this corner case in order to catch it?

Reply:
hmm this is a test class, what do you suggest  to catch?

Reply:
indeed, nevermind.